rules/iam_rules.yaml

```diff
@@ -29,28 +29,28 @@ rules:
           - group:*@{DOMAIN}
 
   # custom rules
-  - name: Allow only service accounts to have access
-    mode: whitelist
-    resource:
-      - type: organization
-        applies_to: self_and_children
-        resource_ids:
-          - '*'
-    inherit_from_parents: true
-    bindings:
-      - role: roles/*
-        members:
-          - serviceAccount:*@*.gserviceaccount.com
+  #- name: Allow only service accounts to have access
```

Comment:
I would like a little more verbiage here to tell the user that these are examples that they can reference.

Reply:
Updated.